java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala

```diff
@@ -323,7 +325,7 @@ class OpenmldbSession {
             }
           } catch {
             case e: Exception => {
-              logger.warn(s"Fail to register table $dbName.$tableName " + ExceptionUtils.getStackTrace(e))
+              logger.warn(s"Fail to register table $dbName.$tableName, exception: " + e.getMessage)
```

Comment:
why revert printing stack? Some messages in our code is hard to understand

Reply:
Sometimes we may not register the online table without offline table info, the trace messages may be misunderstanding. We may handle this NPE first.

Reply:
But when we need to debug why table register failed, the message is not enough. Maybe you can set the trace to debug level. If user meets the register error, user can set log level to get the trace.

Reply:
Okay, I will keep the code of print stacktrace and do not throw exception when registering system tables.